package.json

```diff
@@ -1,27 +1,21 @@
 {
   "name": "news-scraper",
   "version": "1.0.0",
+  "engines": {
+    "node": ">= 16.11.0",
+    "pnpm": ">= 7.0.0"
+  },
   "description": "This app scrapes and cross references daily news from memeorandum.com and mediabiasfactcheck.com and displays it to the user.",
-  "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev-setup:main": "npm install",
-    "dev-setup:frontend": "npm --prefix ./frontend install",
-    "dev-setup:backend": "npm --prefix ./backend install",
-    "dev-setup:scraper": "npm --prefix ./scraper install",
-    "dev-setup:db-migrate": "npm --prefix ./scraper run db:migrate",
-    "dev-setup:all": "npm run dev-setup:main && npm run dev-setup:frontend && npm run dev-setup:backend && npm run dev-setup:scraper && npm run dev-setup:db-migrate",
-    "dev-start:backend": "npm --prefix ./backend run dev",
-    "dev-start:frontend": "npm --prefix ./frontend run start",
+    "preinstall": "npx only-allow pnpm",
+    "dev-setup": "pnpm i && npx nx db-migrate scraper",
+    "dev-start:backend": "npx nx dev backend",
+    "dev-start:frontend": "npx nx start frontend",
     "dev-watch": "concurrently \"npm run dev-start:backend\" \"npm run dev-start:frontend\"",
-    "ci:main": "npm install",
```

Comment:
`ci` is different from `install` command. Also not really sure why these were all added.